### PR TITLE
okx: update parseOrder

### PR DESCRIPTION
**Pull request by sc0Vu**
ccxt/ccxt#15618

Not sure wether we need tpOrdPx or slOrdPx.

### CODE REVIEW

File: js/okx.js [hunk 1]
```diff
@@ -2442,7 +2442,7 @@ module.exports = class okx extends Exchange {
         const marketId = this.safeString (order, 'instId');
         const symbol = this.safeSymbol (marketId, market, '-');
         const filled = this.safeString (order, 'accFillSz');
-        const price = this.safeString2 (order, 'px', 'ordPx');
+        const price = this.safeStringN (order, [ 'px', 'ordPx', 'slOrdPx', 'tpOrdPx' ]);
```
Comment (carlosmiei) [inline js/okx.js:2445]: @sc0Vu can we have `px` and `slOrderPx`/`tpOrderPx` simultaneously?

Comment (sc0Vu) [reply js/okx.js:2445]: Do you mean they'll exist in the same order? Not sure about this, didn't see in their documentation.

Comment (carlosmiei) [reply js/okx.js:2445]: @sc0Vu yes, we have to make sure that we will not "hide" any value

Comment (xmatthias) [reply js/okx.js:2445]: in my understanding, `tpOrdPx` is only set for OCO orders - and is the take profit price (in such a case, the order would have 4 prices).

Stop order (conditional)
image

(OCO order - type "oco")
image

Comment (sc0Vu) [reply js/okx.js]: You're right. We should set these value for stopPrice.

**Pull request closed without merging**